CHANGES.rst

```diff
@@ -9,6 +9,16 @@
 Changes
 =======
 
+Version 2.0.0
```

Comment:
do we need a release already ? or you kept it as a changelog?

Reply:
Mostly as a changelog – until yesterday, the commit was still prefixed with "WIP"... Alternatively, we could drop that commit and track the changes somewhere in an issue.